src/e2e-test/features/pipeline.triggers.feature

```diff
@@ -79,3 +79,28 @@ Feature: Pipeline Triggers
     Then Cleanup pipeline "trigger_test_pipeline_1"
     Then Cleanup pipeline "trigger_test_pipeline_2"
     Then Cleanup pipeline "trigger_test_pipeline_3"
+
+    @PIPELINE_TRIGGERS_TEST
+    Scenario: Deploy seven pipelines and check pagination on inbound triggers tab
+      When Deploy pipeline "trigger_test_pipeline_1" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_2" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_3" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_4" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_5" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_6" with pipeline JSON file "pipeline_with_macros.json"
+      When Deploy pipeline "trigger_test_pipeline_7" with pipeline JSON file "pipeline_with_macros.json"
```

Comment:
Can we use the same list-based notation for above repeated steps with different inputs same as followed for "Open inbound triggers" step?  The same for clean pipeline step as well? Please check once